drivers/hmis/lib/form_data/default/records/move_in_date.json

```diff
@@ -0,0 +1,25 @@
+{
+  "item": [
+    {
+      "link_id": "3.20",
```

Comment:
Doing these as individual mini-forms rather than custom mutations so that they can be overridden per installation / project /etc. For example someone might want to collect a custom note alongside the move-in date.